tests/users/test_register_api.py

```diff
@@ -0,0 +1,20 @@
+from users.models import User
+from rest_framework.test import APITestCase
+from rest_framework import status
+
```

Comment:
Great to see the test case in the expected way. Here are few more things you can test:
1. correct email format +ve/-ve 
1. missing compulsary fields generate error
1. missing optional fields generate no error
1. password length

Reply:
While testing I found the following problems:
1. `__init__.py` file is not in the folder so the test not being executed
1. prefix with `tests_` instead of `test`
1. Actually the test you have written is failing. Please look into it.

Reply:
I've added the error messages, though I'm unsure about whether that was the expected message because now I'm facing difficulties in figuring out what and how to exactly write the test cases.. 😅

Reply:
https://github.com/anitab-org/mentorship-backend/tree/develop/tests/tasks 

You can refer this repo of Anitab about testing. Even though they use flask instead of django. The basic concept is the same just libraries will change.

If you still have more doubts regarding testing we can have a detailed discussion in the slack. Even we can have a video call with the whole backend team to introduce you to testing.

Reply:
Lets remove testing out of this issue. We can create a new issue to perform **testing for registration API**.